amundsen_application/static/js/components/DashboardPage/index.spec.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import * as History from 'history';
+
+import { shallow } from 'enzyme';
+
+import { DashboardPage, DashboardPageProps, RouteProps } from './';
+import { getMockRouterProps } from '../../fixtures/mockRouter';
```

Comment:
Does typescript prefer to use relative import vs absolute import?

Reply:
It can be either. However our build configuration allows absolute imports for the main top level directories to prevent having to do '../../../..' a lot. So line 7 could actually be 
`import { getMockRouterProps } from '../../fixtures/mockRouter';`